homeassistant/components/nest/camera_sdm.py

```diff
@@ -19,6 +21,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# Every 20 seconds, check if the stream is within 1 minute of expiration
```

Comment:
Is this the right approach, or should we instead register a listener when we fetch a new stream that triggers 1 minute prior to expiration?